Sources/Segment/Configuration.swift

```diff
@@ -10,6 +10,15 @@ import Foundation
 import FoundationNetworking
 #endif
 
+// MARK: - Operating Mode
+/// Specifies the operating mode/context
+public enum OperatingMode {
+    /// Running as an executable on a server
+    case server
+    /// Running as a long-lived application
```

Comment:
One could argue a server is longer-lived than an app! stateless might be a better word.

The other thing I'm thinking of is, technically you might want to do async in a server too. At least in other languages I have used for server side.

Reply:
I think async/sync make it clearer what to expect when you switch!